cg/apps/demultiplex/sample_sheet/sample_models.py

```diff
@@ -17,8 +17,8 @@
 LOG = logging.getLogger(__name__)
 
 
-class FlowCellSample(BaseModel):
-    """Class that represents a flow cell sample."""
+class IlluminaIndexSettings(BaseModel):
```

Comment:
Should it not be singular, so it reads:
` IlluminaIndexSetting.lane`

as you access each setting individually (usually)